lib/archethic_web/templates/layout/root.html.eex

```diff
@@ -14,7 +14,8 @@
       <nav class="navbar is-transparent" role="navigation" aria-label="main navigation">
         <div class="navbar-brand">
           <a class="navbar-item is-size-5 is-uppercase" href="<%= Routes.live_path(@conn, ArchethicWeb.ExplorerIndexLive) %>">
-            Archethic
+            <img src="https://raw.githubusercontent.com/archethic-foundation/archethic-assets/main/assets/logos/references_svg/Archethic%20Public%20Blockchain%20-%20White.svg" alt="archethic_logo"/>
+            <%# <img src="<%= Routes.static_path(@conn, "/logo_full_white.svg") >" alt="archethic_logo"/> %>
```

Comment:
In the PR you have added the image, but you're not using it

Reply:
I found the issue related to the image not rendered.
In the https://github.com/archethic-foundation/archethic-node/blob/0c9b18d5ba065541ac31cafadcdd30462b33e24a/lib/archethic_web/endpoint.ex#L31, we only allow the static in this folder so just move to `images` folder should make it

Reply:
Thanks Sam, That helped added to copy assets from images to priv/images in webpack config and it worked.